### ISSUE TITLE
[idea] Erin for Photos ?

### ISSUE THREAD

**Issue by accforgithubtest** [will-moss/erin#11]
(I am not a developer, but have been learning docker basics for self-hosting purposes. So pls excuse me if i am way off with my thinking here.)

Going thru the readme file, it looks like erin is built on top of caddy file server with UI to play/control the videos.

Can there also be a photos feature in erin itself or would that need to be a separate application ? 
I found [caddy imagefilter](github.com/ueffel/caddy-imagefilter), but it doesn't look like a application, but just part of caddy server without ui and cannot display photos but only resize them ?

I am thinking of basically another "photo-wall" type app/ feature that can -

1. Use existing folder based structure and create a photo-wall from all images in the folder and nested subfolders.
2. A Photowall feature that looks similar to something [like this](https://github.com/waschinski/photo-stream). Just an example that looks clean and modern where its not just a fixed sized grid layout, but a better looking tiles of different sizes.
3. A Photoframe/Slideshow/Kiosk feature which just auto-play the photos full screen and width.

I am a big fan of the light weight application and features of both erin and isiah.
Isiah kind of replaced portainer for me for basic docker control / monitoring. 

I am hoping this type of photowall and photoframe feature in a light weight app like erin will probably be a good self hosted app for photo viewing. 
(I currently have photoprism and immich self-hosted and they are heavy for just photo-viewing purposes).

**Comment by accforgithubtest** [comment]
You are right with the questions on how photowall would wor for both photos and videos together. I was having similar questions myself as well if the photo wall feature might not fit in well within erin, given it needs a screen for displaying the photowall first, then another to display the selected photo, whereas erin currently directly starts with the vertical scrolling videos,as is the purpose of the application. 
This photos wall might be better off as a separate application. While I agree that there already exists the photo-stream project which provides similar features already, personally I would love to see a light-weight photowall application on top of caddy, with just the additional UI for photowall, and slideshow features.

For supporting photos within erin, would it be easier to just adopt a mixed mode for both mobile and desktop, displaying photos as well as videos as just vertical scrolling content. Everything in erin stays as-is for videos. When its a photo, erin just displays a very tiny icon somewhere on the UI that its a photo, and auto forwards to the next video / photo after 'x' seconds ? There is already a indicator that says next photo/video coming up after x seconds.

**Comment by will-moss** [comment]
Alright, thanks a ton for your input and insights. 

I'll try to summarize and ask for your confirmation one last time before jumping the gun.

Photo mode will be implemented with the following characteristics:
- Videos and Photos will be mixed together, presented in the existing vertical scrolling feed, as is the case for videos.
- Photos will be shown only for X seconds, where X is an environment variable, set at 5 seconds by default.
- The blue time tracker at the top will also work for Photos.
- Once a photo is "fully shown" (X seconds have passed), the app will auto-scroll to the next media, as is the case currently.

Would that work for you and meet your needs?

**Comment by accforgithubtest** [comment]
Yes, all that sounds perfect. Thanks & Cheers !
